opentracing/src/main/java/opentracing/Tracer.java

```diff
@@ -51,9 +50,44 @@
    *    the operation name for the Span it’s about to create, and
```

Comment:
... and it would be wise for the comment to point out that the operation name should be set on the builder if it's available at `join`-time.

I am still hedging about whether the operation name should be a nullable param here, but it's cleaner to leave it off for now, I think. (I'm also having some doubts about what we've done in other languages, esp python where `operation_name` could be an optional named parameter... @yurishkuro thoughts?)